packages/customWidgets/calendar-web/tests/e2e/specs/renderDifferentViews.spec.ts

```diff
@@ -24,4 +24,10 @@ describe("Calendar", () => {
         calendar.calendarSlot.waitForExist();
         expect(calendar.daysDisplayed).toBe(1);
     });
+    it("renders start date attribute correctly", () => {
+        page.open("p/startPosition");
+        const calendar = new Calendar("calendar2");
+        calendar.element.waitForExist();
+        expect(calendar.label.getText()).toBe("Tuesday Feb 02");
```

Comment:
This test will not verify correct behaviour on every 2nd of February each year. Can you adjust the label in the widget config in such a way that the year is printed as well. If we use this year, we know the date has already passed. This is important, because by default the calendar will open on the current day.

Reply:
Yeah, I can do that. But the next February 2 on Tuesday will be in 2027 I wasn't expecting this test to fail until there. 
😂

Reply:
Done! 😁